src/main/java/kotlinx/reflect/lite/KDeclarationContainer.kt

```diff
@@ -0,0 +1,14 @@
+package kotlinx.reflect.lite
+
+/**
+ * Represents an entity which may contain declarations of any other entities,
+ * such as a class or a package.
+ */
+interface KDeclarationContainer {
+    /**
+     * All functions and properties accessible in this container.
+     * TODO: inherited, static and extension members are not supported yet
+     * NOTE: kotlinx-metadata-jvm can not provide inherited and static members - it's a low level api
```

Comment:
Could you provide some examples of unsupported behavior that is supported in the reflect library? In particular, I don't understand the part about extension members. I think it's impossible for the reflect library to provide information about `X.y` when querying `X` for its members if `y` is an extension, because extensions being or not being available depends on the current imports. Do you mean something else by this?